packages/components/src/DataList/DataList.tsx

```diff
@@ -55,5 +64,32 @@ export function DataList<T extends DataListObject>({
   );
 }
 
+function ResultsCounter({
+  totalCount,
+  loading,
+}: {
+  totalCount: number | null | undefined;
+  loading: boolean;
+}) {
+  if (totalCount === undefined) return null;
+  let output = null;
+
+  if (totalCount === null && loading) {
+    output = <Glimmer size="auto" shape="rectangle" />;
+  }
+
+  if (typeof totalCount === "number") {
+    output = (
+      <Text variation="subdued">({totalCount.toLocaleString()} results)</Text>
+    );
+  }
+
+  return (
+    <div className={styles.results} data-testid="results">
```

Comment:
This isn't really broadcasted but i'd like to make that testID more unique and specific to Atlantis. 

[Internal post about it](https://jobber.atlassian.net/wiki/spaces/TEAMATL/pages/2476933462/How+should+we+declare+test+ID+s+in+Atlantis+components)

Let's think about testing in general. That test ID is too generic that someone outside of atlantis would easily write a testID called "results" in their component. They then get confused as to why jest is complaining about finding 2 elements with that test ID. 😆